models/organization/org.go

```diff
@@ -400,6 +400,7 @@ func DeleteOrganization(ctx context.Context, org *Organization) error {
 		&TeamUnit{OrgID: org.ID},
 		&TeamInvite{OrgID: org.ID},
 		&secret_model.Secret{OwnerID: org.ID},
+		&user_model.Blocking{BlockerID: org.ID},
```

Comment:
And `&user_model.Blocking{BlockeeID: org.ID},`?

Reply:
You can't block an organization.

Reply:
Wait, why?
That seems like a pretty arbitrary decision to me.
In other words, a spammer can simply do everything under an org, and thus prevent being blocked?

Reply:
No, if I'm not mistaken you can't do anything as an org. You can't login as the org (user), therefore you can't perform any actions as an org.